app/assets/scripts/app.js

```diff
@@ -43,3 +44,14 @@ $(document).ready(function () {
         };
 })(jQuery);
 
+// mobile dropdown menu state change 
+var currentState = []
+function changeMenu(e) {
```

Comment:
this code is pretty opaque, can you please leave a comment about what it is doing?